packages/bitcore-client/src/wallet.ts

```diff
@@ -122,7 +122,12 @@ export class Wallet {
     if (alreadyExists) {
       throw new Error('Wallet already exists');
     }
-    const wallet = Object.assign(params, {
+    const wallet = Object.assign( {
```

Comment:
We could remove Object.assign completely

Reply:
sorry I didn't see this until too late, maybe I can remove it in the future